workers/data_refinery_workers/processors/transcriptome_index.py

```diff
@@ -33,39 +45,31 @@ def _prepare_files(job_context: Dict) -> Dict:
     """
     # Transcriptome index processor jobs have only one batch. Each
     # batch has a fasta file and a gtf file
-    batch = job_context["batches"][0]
-    fasta_file = File.objects.get(batch=batch, raw_format__exact="fa.gz")
-    gtf_file = File.objects.get(batch=batch, raw_format__exact="gtf.gz")
-
-    try:
-        fasta_file.download_raw_file(job_context["job_dir_prefix"])
-        gtf_file.download_raw_file(job_context["job_dir_prefix"])
-    except Exception:
-        logger.exception("Exception caught while retrieving raw files.",
-                         processor_job=job_context["job_id"],
-                         batch=batch.id)
-
-        job_context["job"].failure_reason = "Exception caught while retrieving raw files."
-        job_context["success"] = False
-        return job_context
-
-    # The files are gzipped when download, but rsem-prepare-reference
-    # expects them to be unzipped.
-    gzipped_fasta_file_path = fasta_file.get_temp_pre_path(job_context["job_dir_prefix"])
-    gzipped_gtf_file_path = gtf_file.get_temp_pre_path(job_context["job_dir_prefix"])
-    job_context["fasta_file_path"] = gzipped_fasta_file_path.replace(".gz", "")
-    job_context["gtf_file_path"] = gzipped_gtf_file_path.replace(".gz", "")
-
-    with gzip.open(gzipped_fasta_file_path, "rb") as gzipped_file, \
-            open(job_context["fasta_file_path"], "wb") as gunzipped_file:
-        shutil.copyfileobj(gzipped_file, gunzipped_file)
+    #batch = job_context["batches"][0]
```

Comment:
I don't think we need this guy any more.

Reply:
Roger